static/src/javascripts/projects/commercial/modules/liveblog-adverts.js

```diff
@@ -7,24 +12,12 @@ import { commercialFeatures } from 'common/modules/commercial/commercial-feature
 import { createSlot } from 'commercial/modules/dfp/create-slot';
 import { spaceFiller } from 'common/modules/article/space-filler';
 
-const INTERVAL = 5; // number of posts between ads
 const OFFSET = 1.5; // ratio of the screen height from which ads are loaded
 const MAX_ADS = 8; // maximum number of ads to display
 
-type spaceFillerRules = {
```

Comment:
Where did these go? Were they duplicated elsewhere?

Reply:
yes, I had previously worked on making a single type for rules. But live blogs had always duplicated their own rules type.